turtleFSI/modules/domain.py

```diff
@@ -48,6 +55,19 @@ def assign_domain_properties(dx, dx_f_id, rho_f, mu_f, fluid_properties, dx_s_id
             solid_properties.append({"dx_s_id":dx_s_id,"material_model":material_model,"rho_s":rho_s,"mu_s":mu_s,"lambda_s":lambda_s})
     elif isinstance(solid_properties, dict): 
         solid_properties = [solid_properties]
+    
+    # RobinBC
+    if robin_bc==True:
+        ds_s = {}
+        if isinstance(ds_s_id, list): # If ds_s_id is a list (i.e, if there are multiple boundary regions):
+            for solid_boundaries in range(len(ds_s_id)):
+                ds_s[solid_boundaries] = ds(ds_s_id[solid_boundaries], subdomain_data=boundaries) # Create ds_s for each boundary
+                ds_s_ext_id_list=ds_s_id
+        else:
+            ds_s[0] = ds(ds_s_id, subdomain_data=boundaries)
+            ds_s_ext_id_list=[ds_s_id] # If there aren't multpile boundary regions, and the boundary parameters are given as floats, convert solid parameters to lists.
+    else:
+        ds_s = None
+        ds_s_ext_id_list = None
```

Comment:
What is `ds_s_id` if it is not a list?

Reply:
I think `ds_s_id` could be a single integer if it’s not a list, and we do not need to return the `ds_s_id` because it is just used to create `ds_s` and `ds_s_ext_id_list`. As I’m not the main author of `domain.py`, I may not understand the whole functionality of this file... Maybe good to discuss with @dbruneau-mie